catalog/core/catalog-core-commons/src/test/java/ddf/catalog/filter/delegate/ValidationQueryDelegateTest.java

```diff
@@ -572,12 +436,14 @@ public void testPropertyIsBetweenObject() throws UnsupportedQueryException {
 
     @Test
     public void testPropertyIsNullTrueValidationErrors() throws UnsupportedQueryException {
-        assertThat(testValidationQueryDelegate.propertyIsNull(VALIDATION_ERRORS), is(true));
+        assertThat(testValidationQueryDelegate.propertyIsNull(BasicTypes.VALIDATION_ERRORS),
+                is(true));
     }
 
     @Test
     public void testPropertyIsNullTrueValidationWarnings() throws UnsupportedQueryException {
-        assertThat(testValidationQueryDelegate.propertyIsNull(VALIDATION_WARNINGS), is(true));
+        assertThat(testValidationQueryDelegate.propertyIsNull(BasicTypes.VALIDATION_WARNINGS),
```

Comment:
I think there are, but they don't show up in the diff. For example, the test after this two is negative. Let me know if that's not what you mean.